ui/src/dev-server.js

```diff
@@ -0,0 +1,40 @@
+// @flow
+
+const express = require('express');
+const webpack = require('webpack');
+const webpackDevMiddleware = require('webpack-dev-middleware');
+const webpackHotMiddleware = require('webpack-hot-middleware');
+const webpackConfig = require('../__config__/webpack.config.dev')({
+    browser: true,
+});
+
+const compiler = webpack(webpackConfig);
+const app = express();
+
+app.use(
+    webpackDevMiddleware(compiler, {
+        publicPath: webpackConfig.output.publicPath,
+        noInfo: true,
+    })
+);
+app.use(webpackHotMiddleware(compiler));
+app.get('/', (req, res) => {
+    delete require.cache[require.resolve('../dist/ui.bundle.server')];
+
+    // eslint-disable-next-line global-require
+    const { frontend } = require('../dist/ui.bundle.server');
+
+    res.send(
+        frontend.render({
```

Comment:
Add a `TODO` to pass the props coming along side the `req` rather than hard-coding them.
Unless you want to add this in this PR